package.json

```diff
@@ -57,6 +64,7 @@
     "format-fix": "prettier --write \"src/**/*.js\"",
     "start": "node --require esm make dev -s -w -d data",
     "build": "npm run lib",
```

Comment:
If I run this target, I get an error....

Error during execution of Command  custom
Error: Unexpected character '@' (Note that you need plugins to import files that are not JavaScript)

Now, if this target is still going to hang around for a bit then it would be nice to fix it!